client_connection/manager.go

```diff
@@ -39,12 +39,12 @@ func NewManager(mysteriumClient server.Client, dialogEstablisherFactory DialogEs
 }
 
 func (manager *connectionManager) Connect(identity identity.Identity, NodeKey string) error {
-	manager.status = statusConnecting()
 	clientSession, err := manager.mysteriumClient.SessionCreate(NodeKey)
 	if err != nil {
 		manager.status = statusError(err)
 		return err
 	}
+	manager.status = statusConnecting(clientSession.Id)
```

Comment:
First of all sessionid comming from discovery service will be removed in the future. There is no point exposing it when status is connecting as a lot of can happen even before connection is really established. There fore status connecting WITHOUT session id was done intentionally. @Waldz your word is needed here.

Reply:
- Just dont use variable from endpoint `mysteriumClient.SessionCreate()`
- Status "connecting" should start in very beggining  of `Connect()`